test/tab-components/tabTest.js

```diff
@@ -390,6 +390,16 @@ describe('tab tests', function () {
         .waitForExist('.frameWrapper.isPreview webview[data-frame-key="2"]')
         .moveToObject(urlInput)
     })
+    it('does not show preview in the active tab', function * () {
+      yield this.app.client
+        .moveToObject('[data-test-id="tab"][data-frame-key="2"]')
+        .waitForTextValue('[data-test-id="tab"][data-frame-key="2"]', 'Page 1')
+        .waitForExist('.frameWrapper.isPreview webview[data-frame-key="2"]')
+        .moveToObject('[data-test-id="tab"][data-frame-key="1"]')
+        .click('[data-test-id="tab"][data-frame-key="1"]')
+        .waitForExist('[data-test-active-tab][data-frame-key="1"]')
+        .waitForExist('.frameWrapper.isPreview webview[data-frame-key="2"]', 500, true)
```

Comment:
I am not sure that I understand this test. You move mouse into frame 1, you click on it, then you wait for frame 1 to be active and then you check if frame 2 is in preview?